docs/reference/search/search-your-data/search-across-clusters.asciidoc

```diff
@@ -290,14 +290,25 @@ means the document came from the local cluster.
 [[skip-unavailable-clusters]]
 === Skip unavailable clusters
 
-By default, a {ccs} returns an error if *any* cluster in the request is
-unavailable.
+By default, a {ccs} request fails if a remote cluster in the request is
+unavailable. Use the <<skip-unavailable,`skip_unavailable`>> cluster setting to
+mark a specific remote cluster as optional in cross-cluster searches.
 
-To skip an unavailable cluster during a {ccs}, set the
-<<skip-unavailable,`skip_unavailable`>> cluster setting to `true`.
+If `skip_unavailable` is `true`, a {ccs}:
 
-The following <<cluster-update-settings,cluster update settings>> API request
-changes `cluster_two`'s `skip_unavailable` setting to `true`.
+* Skips the remote cluster if its nodes are unavailable during the search. The
+response's `_cluster.skipped` value contains a count of any skipped clusters.
+
+* Ignores errors, such as errors related to unavailable shards or indices,
+returned by the remote cluster.
+
+* Ignores the <<search-partial-responses,`allow_partial_search_results`>>
+parameter and the related `search.default_allow_partial_results` cluster setting
+when searching the remote cluster. This means searches on the remote cluster may
```

Comment:
I think it's good to mention partial results. There are other settings like allow_no_indices and ignore_unavailable that may lead to errors getting ignored, but I wonder if we have to list them all. If the cluster is made optional, any conflicting configuration that normally leads to errors is superseded by skip_unavailable

Reply:
I added this info to the second bullet point:

> If `skip_unavailable` is `true`, a CCS:
> * ...
> * Ignores errors, such as errors related to unavailable shards or indices,
returned by the remote cluster. This can include errors related to search
parameters such as `allow_no_indices` and `ignore_unavailable`.